homeassistant/components/ai_speaker/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""Support for AI-Speaker."""
+import logging
+
+from aisapi.ws import AisWebService
+
+from homeassistant.helpers import aiohttp_client
+
+from .const import DOMAIN
+
+PLATFORMS = ["sensor", "media_player"]
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass, config):
+    """Set up the initial domain configuration."""
+
+    async def async_command(service):
+        """Publish command to AI-Speaker WS."""
+        web_session = aiohttp_client.async_get_clientsession(hass)
+        ais_url = service.data["ais_url"]
+        ais_gate = AisWebService(hass.loop, web_session, ais_url)
+        await ais_gate.command(service.data["key"], service.data["val"])
+
+    hass.services.async_register(DOMAIN, "publish_command", async_command)
```

Comment:
A service is only needed if we want to allow users direct access to the web service. Please don't add the service in this first PR.

Reply:
ok, done in commit @asystentka-jolka
improvements after first review
6214ef8